pkg/store/prometheus.go

```diff
@@ -195,11 +196,13 @@ func (p *PrometheusStore) Series(r *storepb.SeriesRequest, s storepb.Store_Serie
 		return nil
 	}
 
-	if r.QueryHints != nil && r.QueryHints.IsSafeToExecute() {
+	shardMatcher := r.ShardInfo.Matcher()
+	if r.QueryHints != nil && r.QueryHints.IsSafeToExecute() && !shardMatcher.IsSharded() {
 		return p.queryPrometheus(s, r)
 	}
 
 	q := &prompb.Query{StartTimestampMs: r.MinTime, EndTimestampMs: r.MaxTime}
+	level.Info(p.logger).Log("min_time", r.MinTime, "max_time", r.MaxTime)
```

Comment:
This log will be printed a lot of times. Can we add more fields and make it debug level?

Reply:
Yeah this was only for debugging purposes :) Should be gone now.

Reply:
It's still here? :thinking:

Reply:
Should be gone for good :)